src/main/webapp/index.html

```diff
@@ -3,9 +3,18 @@
   <head>
     <meta charset="UTF-8">
     <title>Play User Data Beetle</title>
+    <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getDatabases()">
     <h1>Play User Data Beetle</h1>
     <p>Click <a href="/example-db">here</a> to view data queried from an example Cloud Spanner database.</p>
+    <p>please select the database you would like to use</p>
+
+    <div id="select-db">
+        <select id="list-databases" value="list-databases" class="select">
+            <option id="select-title">Select a Database</option>
+        </select>
+    </div>
```

Comment:
Should we start with a "Loading..." message until the list is retrieved?

Reply:
I like that idea too!

Reply:
Done!